defaults/main.yml

```diff
@@ -17,8 +17,11 @@ orientdb_enable_http: True
 orientdb_http_port_range: 2480-2490
 orientdb_force_install: False
 orientdb_users:
-- name: root
-  password: root
+  - name: root
+    password: root
+  - name: guest
+    password: guest
+    resources: 'connect,server.listDatabases,server.dblist'
```

Comment:
I have copied from static guest user definition. From documentation was able to find 

```
server.listDatabases - Lists available databases on the server.
```

I guess ```server.dblist``` means ```list databases``` too but it is deprecated and left for compatibility reasons